src/components/ButtonCheck.spec.js

```diff
@@ -0,0 +1,22 @@
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import ButtonCheck from './ButtonCheck';
+
+describe('ButtonCheck', () => {
+  it('renders a button', () => {
+    const onClick = jest.fn();
+    render(<ButtonCheck onClick={onClick} isActive={false} />);
+
+    const button = screen.getByRole('button');
+    expect(button).toBeInTheDocument();
+  });
+
+  it('has onClick function', () => {
+    const onClick = jest.fn();
+    render(<ButtonCheck onClick={onClick} isActive={false} />);
+
+    const button = screen.getByRole('button');
+    userEvent.click(button);
+    expect(onClick).toHaveBeenCalled();
+  });
+});
```

Comment:
There are two tests missing: if isActive is true, then the name of the button is 'gesehen', if it is false, then it's 'ungesehen'.